OpenRA.Game/Actor.cs

```diff
@@ -295,6 +295,11 @@ public bool IsDisabled()
 			return false;
 		}
 
+		public bool IsInvisibleTo(Player player)
```

Comment:
Or perhaps better - just inline this where you need it.  This is a special case workaround, not a general-use function.